src/network/axiosClient.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import { errorHandler } from "../utils/utils";
+
+const axiosClient = axios.create();
+
+axiosClient.defaults.baseURL = `${import.meta.env.VITE_BASE_URL}`;
+
+axiosClient.interceptors.response.use(function (response) {
```

Comment:
prefer arrow functions, and better to move errorHandle logic here, one-liner functions aren't recommended